core/trino-server-rpm/src/main/resources/dist/etc/init.d/trino

```diff
@@ -37,6 +37,16 @@ CONFIGURATION=(
     --server-log-file "${SERVER_LOG_FILE:-/var/log/trino/server.log}"
 )
 
+files_limit=131072
+if current_limit=$(ulimit -Hn) && [ "$files_limit" -lt "$current_limit" ]; then
+    files_limit=$current_limit
```

Comment:
nit:  "brace" and quote variables, like in `"${var}"`

Reply:
Done!